2-mandatory/5-eligible-students.js

```diff
@@ -6,9 +6,15 @@
     (see tests to confirm how this data will be structured)
   - Returns an array containing only the names of the who have attended AT LEAST 8 classes
 */
-
-function getEligibleStudents() {}
-
+function getEligibleStudents(studentsData) {
+  let canDoExam = [];
+  for(let i = 0; i < studentsData.length; i++){
+    if(studentsData[i][1] >= 8){
+      canDoExam.push(studentsData[i][0]);
+    }
+  }
+  return canDoExam;
```

Comment:
good use of variable names again, a for loop works well, another approach could be to use `.filter()` to get the eligible students, followed by a `.map()` to get the first name :)